guacamole-docker/bin/build-guacamole.sh

```diff
@@ -41,6 +41,7 @@
 
 BUILD_DIR="$1"
 DESTINATION="$2"
+BUILD_PROFILE="$3"
```

Comment:
This new `BUILD_PROFILE` parameter for `build-guacamole.sh` needs to be documented with an `@param` in the corresponding comment block (see the Doxygen-style block above covering this script and the other two parameters).

Reply:
added the parameter documentation as requested